feature/matcher/src/main/java/com/simprints/matcher/usecases/FaceMatcherUseCase.kt

```diff
@@ -43,7 +51,11 @@ internal class FaceMatcherUseCase @Inject constructor(
         val resultItems = createRanges(totalCandidates)
             .map { range ->
                 async(dispatcher) {
-                    val batchCandidates = getCandidates(matchParams.queryForCandidates, range, dataSource = matchParams.biometricDataSource)
+                    val batchCandidates = getCandidates(
+                        matchParams.queryForCandidates,
```

Comment:
I think `queryWithSupportedFormat` should also be used here

Reply:
Maybe also add a test to ensure this does not regress by mistake in the future.

Reply:
@BurningAXE Done

Reply:
Looks like `matchParams.queryForCandidates` is still used instead of `queryWithSupportedFormat`

Reply:
It appears to be a glitch in GitHub's diff view. You can check out the code to see that the change has already been made.
![image](https://github.com/user-attachments/assets/f0039773-3270-4c02-81b8-677fa60e68b1)